grayskull/strategy/pypi.py

```diff
@@ -582,6 +585,21 @@ def check_noarch_python_for_new_deps(
     config.is_arch = False
 
 
+def set_python_min(req_list: list, section: str) -> list:
+    if not req_list:
+        return req_list
+    python_min = "<{ python_min }}"
+    map_section = {
+        "host": f"{python_min}.*",
+        "run": f">={python_min}",
+        "test": f"={python_min}",
```

Comment:
These changed recently.

```suggestion
        "host": f"{python_min}",
        "run": f">={python_min}",
        "test": f"{python_min}",
```